website/blog/2022-10-19-polyglot-dbt-python-dataframes-and-sql.md

```diff
@@ -0,0 +1,347 @@
+1. **Python workflows where your data lives**: Most Python work being done today is done outside of the data platform that stores the source data, meaning developers have to transfer the data into the Python environment first, which adds significant friction to the development workflow. In the case of Snowpark DataFrames, data movement is reduced by pushing the computation down to the data platform.
+
+## Trade-offs between SQL and dataframe syntax
+
+Once you’re set up - the next question is: what _should_ you use Python for? How should you think about tradeoffs between SQL vs. dataframes? I haven’t personally pondered this deeply… but the folks at [Ponder](https://ponder.io/) have 😉 They published a series of posts comparing the two in terms of:
+
+- [convenience](https://ponder.io/pandas-vs-sql-part-4-pandas-is-more-convenient/) - dataframes go well with data science libraries like [Scikit-learn](https://scikit-learn.org/stable/)
+- [conciseness](https://ponder.io/pandas-vs-sql-part-2-pandas-is-more-concise/) - dataframes have concise syntax for normalization, one-hot encoding, rolling averages, and other uses
+- [flexibility](https://ponder.io/pandas-vs-sql-part-3-pandas-is-more-flexible/) - rows and columns in dataframes can be transposed and functions can be applied along columns or rows
+
+Gaining **your own** sense of these differences will empower you to create your own alloy mix of polyglot dbt models.
+
+## Comparing dataframe libraries
+
+Before we get into our hands-on example, let’s take a look at the nuts and bolts of getting your project working with different dataframe types. Multiple data platforms and dataframe libraries are supported in dbt Core as of version 1.3, but not uniformly (see compatibility table below). See [here](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/python-models#specific-data-platforms) for platform-specific setup instructions.
+
+| **Type of data frame**     | **Snowflake**                      | **Databricks**                    | **BigQuery**  |
+|----------------------------|------------------------------------|-----------------------------------|---------------|
+| Snowpark DataFrames        | ✅                                  | ❌                                 | ❌             |
+| pandas DataFrames          | ✅                                  | ✅                                 | ✅             |
+| Spark DataFrames           | ❌                                  | ✅                                 | ✅             |
+| pandas-on-Spark DataFrames | ❌                                  | ✅                                 | ✅             |
+| Koalas DataFrames          | ❌                                  | ✅                                 | ✅             |
```

Comment:
this could be confusing, but is technically correct. I don't think we need to change it, just as a callout